### DISCUSSION TITLE
What's wrong?

### DISCUSSION THREAD

**Discussion by ghost** [AntlerVC/firetable#1220] [Support / Q&A] [answered]
I have deleted the whole workspace and the project and still get the issue
![Screenshot 2023-04-21 at 5 14 56 pm](https://user-images.githubusercontent.com/92528607/233567590-4c212811-d4e9-49fa-ad7e-cfdb9ab56184.png)

**Comment by BeeBombshell** [comment, marked as answer]
Hey @booksports, are you trying to recreate a deleted project? Could you follow the steps on [this](https://docs.rowy.io/faqs/deleted-project) docs page so that we can help you delete the project from our end? For the 4th step, you can drop a message with your project-id here or email us at support@rowy.io.
